static/app/components/globalDrawer/index.tsx

```diff
@@ -0,0 +1,109 @@
+import {createContext, useCallback, useContext, useEffect, useRef, useState} from 'react';
+import {AnimatePresence} from 'framer-motion';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import {DrawerBody, DrawerPanel} from 'sentry/components/globalDrawer/components';
+import type {
+  DrawerConfig,
+  DrawerContext as TDrawerContext,
```

Comment:
Nit (feel free to dismiss), but I havent seen us use the T conventions for types decl naming